src/main/java/de/uni_tuebingen/ub/nppm/db/SaveHelper.java

```diff
@@ -22,52 +25,72 @@ public static void insertMetaData(String form, int id, int benutzerID, int grupp
         insertOrUpdate(sql);
     }
 
-    public static List<Map<String, String>> selectAttribute(String table, String attribute, Map<String, String> andConditions) throws Exception {
+    public static List<Map<String, String>> getAttribute(String table, String attribute, Map<String, String> andConditions) throws Exception {
```

Comment:
All these get... functions should also be overloaded select... functions as specified in the previous comments. 

I mean, in the end i'm not 100% sure whether the functions should all be called select... or get... but we should name all functions the same that also do similar things and just overload them, instead of having lots of different function names for functions that do basically the same.

Reply:
Me and Julian have tested it, the have diverent return types like List<Map>, String and also:

//Diese funktion ist notwendig für ist<Map> attributes = SaveHelper.getAttributeMap(zieltabelle, zielAttributStr, id); in doSave, da sie ein Single-Objekt zurück gibt
  //  und sonst ein Exception auftritt wenn ich die public static List<Map> getAttribute(String table, String attribute, Map andConditions) throws Exception { benützen würde
    public static List<Map> getAttributeMap(String table, String attributeString, int id) throws Exception {